### ISSUE TITLE
Commands not being available

### ISSUE THREAD

**Issue by NillerMedDild** [Quarris/EnigmaticGraves#9]
Relayed from: https://github.com/NillerMedDild/Enigmatica6/issues/2698

We received a slightly strange bug report ^, the user cannot access all Enigmatic Graves commands, despite him being OP, the issue even persisted when the commands were run from the server console.

I haven't been able to reproduce this, but maybe it's an OS thing?

**Comment by NillerMedDild** [comment]
After

**Comment by Mx772** [comment]
Not sure the cause, but also got this same error. I was actually able to brute-force the grave by doing the following in-game:
/enigmatic_graves <name> restore death_X_ true
where X is a number that I kept incrementing up until it would restore the right one. 

Even on reboot it doesn't work correctly. Same error as them:

```
[13:02:26] [Netty Epoll Server IO #4/ERROR] [minecraft/ArgumentTypes]: Could not serialize dev.maxneedssnacks.interactio.command.RegistryArgument@3459718d (class dev.maxneedssnacks.interactio.command.RegistryArgument) - will not be sent to client!
[13:02:26] [Netty Epoll Server IO #4/ERROR] [minecraft/ArgumentTypes]: Could not serialize dev.quarris.enigmaticgraves.command.GraveEntryType@5ce096f5 (class dev.quarris.enigmaticgraves.command.GraveEntryType) - will not be sent to client!
```

If I do it via terminal connection, it actually freezes the CLI/Terminal. 

Even if another player kills themselves, no list is auto-populated via tab. 

In game, it actually says your command is invalid:
![image](https://user-images.githubusercontent.com/9059161/126038083-ee3f27e6-57c4-45b1-9fc6-7653ad582a8c.png)
![image](https://user-images.githubusercontent.com/9059161/126038103-74398c81-d799-4184-b905-f7e7f2eae7cc.png)

Even though the second one will restore. 

Running server on:
Ubuntu 20.04.1 LTS (GNU/Linux 5.4.0-48-generic x86_64)
openjdk version "16.0.1" 2021-04-20
OpenJDK Runtime Environment (build 16.0.1+9-Ubuntu-120.04)
OpenJDK 64-Bit Server VM (build 16.0.1+9-Ubuntu-120.04, mixed mode, sharing)

Unfortunately on a Server, so cannot exactly wipe the world. 

We can keep this closed for now if you want. But looks like this is possibly still an underlying issue. 
Not sure if there is any debug logs or something you want, but can toss those over.

**Comment by Mx772** [comment]
Prolly be helpful if I mentioned versions:

**Modpack:**
Fresh Install - Enigmatica 6 v0.5.6

**Mod Version:**
EnigmaticGraves-1.3.jar

**Extra Mods on Server:**
[Discord-Integration](https://www.curseforge.com/minecraft/mc-mods/dcintegration) (now disabled, but enabled at server start.)
[mc2discord](https://www.curseforge.com/minecraft/mc-mods/mc2discord/)

**Java Args:**
```
  javaArgs:
    - "-XX:+UseG1GC"
    - "-XX:+ParallelRefProcEnabled"
    - "-XX:MaxGCPauseMillis=200"
    - "-XX:+UnlockExperimentalVMOptions"
    - "-XX:+DisableExplicitGC"
    - "-XX:+AlwaysPreTouch"
    - "-XX:G1NewSizePercent=30"
    - "-XX:G1MaxNewSizePercent=40"
    - "-XX:G1HeapRegionSize=8M"
    - "-XX:G1ReservePercent=20"
    - "-XX:G1HeapWastePercent=5"
    - "-XX:G1MixedGCCountTarget=4"
    - "-XX:InitiatingHeapOccupancyPercent=15"
    - "-XX:G1MixedGCLiveThresholdPercent=90"
    - "-XX:G1RSetUpdatingPauseTimePercent=5"
    - "-XX:SurvivorRatio=32"
    - "-XX:+PerfDisableSharedMem"
    - "-XX:MaxTenuringThreshold=1"
    - "-Dusing.aikars.flags=https://mcflags.emc.gs"
    - "-Daikars.new.flags=true"
    - "-Dfml.readTimeout=180" # servertimeout
    - "-Dfml.queryResult=confirm" # auto /fmlconfirm
    - "--add-opens=java.base/sun.security.util=ALL-UNNAMED"
    - "--add-opens=java.base/java.util.jar=ALL-UNNAMED"
```

**maxRam:**
12G

**Comment by Quarris** [comment]
This should be fixed in v1.4.2
